### PR TITLE
More mutations

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
Quick review please and also, a question:

When I submit a query via GraphiQL and get an error, my error looks like this:
"errors": [
    {
      "locations": [
        {
          "column": 3,
          "line": 2
        }
      ],
      "message": "name: can't be blank",
      "path": [
        "createUser"
      ]
    }
Yet my error type looks like this"
 @desc "Ecto changeset errors from an input"
  object :input_error do
    field :key, :string
    field :message, :string
    field :details, :string
  end

This doesn't seem to add up?!
This is the type I defined before I used Middleware.

Enlighten me, please :)

### CODE REVIEW

**Comment by alanvardy** [comment]
> Quick review please and also, a question:
> 
> When I submit a query via GraphiQL and get an error, my error looks like this: "errors": [ { "locations": [ { "column": 3, "line": 2 } ], "message": "name: can't be blank", "path": [ "createUser" ] } Yet my error type looks like this" @desc "Ecto changeset errors from an input" object :input_error do field :key, :string field :message, :string field :details, :string end
> 
> This doesn't seem to add up?! This is the type I defined before I used Middleware.
> 
> Enlighten me, please :)

The error it returned is standard error for absinthe. I am a bit confused about why a custom "input error" would be desired. Perhaps I could view the video you are referring to.

File: lib/graphql_practice_web/resolvers/user.ex [hunk 1]
```diff
@@ -16,12 +17,32 @@ defmodule GraphqlPracticeWeb.Resolvers.User do
   def create_user(_, params, _) do
     case Accounts.create_user(params) do
       {:error, changeset} ->
-        {:error,
-         message: "Could not create user!",
-         details: Ecto.Changeset.traverse_errors(changeset, fn {msg, _opts} -> msg end)}
+        {:error, changeset}
 
       {:ok, user} ->
         {:ok, %{user: user}}
```
Comment (alanvardy) [inline lib/graphql_practice_web/resolvers/user.ex]: Could put these back to single lines if you want.

**Review by alanvardy — APPROVED** [review]
I think this is really solid.

File: lib/graphql_practice_web/resolvers/user.ex (hunk 1 above)
Comment (CorneliaKelinske) [reply lib/graphql_practice_web/resolvers/user.ex]: I guess :)

**Pull request merged**